### DISCUSSION TITLE
feature: security logs for critical editor changes

### DISCUSSION THREAD

**Discussion by Khaikaa** [AscensionGameDev/Intersect-Engine#2213] [Ideas]
**Is your feature request related to a problem? Please describe.**
There is no logging when it comes to editor-server connections, so it's impossible to track any workflow.

**Describe the solution you'd like**
A serverside log file to store logs of any database modification made by the editor. For example: [2021-02-26 12:36] **Khaikaa** made changes on **common event <name/id>**

**Additional context**
I don't think we need super detailed logs, but it would be great to log when someone makes changes on the server through the editor, specially for projects developed by a team. This way you can check when someone did something, and which kind of action that person did.

**Comment by blinkuz** [comment]
Continuing with the forum conversation, the idea of registering even changes at the administrative level not only of events can be a good solution, the ideal would be to identify those processes as you mentioned before in the forum comment changes of power, mute, ban and Other important actions done from different means like events / the server console or the api should be logged.
